botcraft/src/AI/Tasks/InventoryTasks.cpp

```diff
@@ -325,19 +325,58 @@ namespace Botcraft
             Position(1, 0, 0), Position(-1, 0, 0)
             });
 
-        // Check if block is air
         bool midair_placing = true;
+        // If no face specified
+        if (!face.has_value())
+        {
+            if (allow_midair_placing) // Then we don't care and default to Up
+            {
+                face = PlayerDiggingFace::Up;
```

Comment:
You can't change directly the value of face as it's declared const in the hpp. Either use an intermediary variable or change the const in the function parameters.